src/test/java/jpa/domain/MemberRepositoryTest.java

```diff
@@ -54,4 +62,53 @@ public void findByName() {
         // then
         assertThat(findMember.get().getId()).isEqualTo(memberId);
     }
+
+    @Test
+    @DisplayName("사용자에서 즐겨찾기들 조회 테스트")
+    public void findMappedFavorite() {
+        // given
+        Member member = new Member();
+        Favorite favorite1 = new Favorite();
+        Favorite favorite2 = new Favorite();
+        member.addFavorites(this.favoriteRepository.save(favorite1));
+        member.addFavorites(this.favoriteRepository.save(favorite2));
+        Member savedMember = this.memberRepository.save(member);
+
+        // when
+        Optional<Member> memberOptional = this.memberRepository.findById(savedMember.getId());
+        List<Favorite> favorites = memberOptional.get().getFavorites();
+
+        // then
+        assertAll(
+                () -> assertThat(favorites.size()).isEqualTo(2)
+                , () -> assertThat(favorites.toString()).contains(favorite1.getId() + "", favorite2.getId() + "")
+        );
+    }
+
+    @Test
+    @DisplayName("사용자 ID로 사용자에 등록 된 즐겨찾기의 출발역 조회 테스트")
+    public void findStation() {
+        // given
+        Member member = new Member();
+        String stationName1 = "강남역";
+        String stationName2 = "교대역";
+        String regex = "^[" + stationName1 + "|" + stationName2 + "]";
+        Station station1 = this.stationRepository.save(new Station(stationName1));
+        Station station2 = this.stationRepository.save(new Station(stationName2));
+        Favorite favorite1 = new Favorite(station1, station2);
+        Favorite favorite2 = new Favorite(station2, station1);
+        member.addFavorites(this.favoriteRepository.save(favorite1));
+        member.addFavorites(this.favoriteRepository.save(favorite2));
+        Member savedMember = this.memberRepository.save(member);
+
+        // when
+        Optional<Member> memberOptional = this.memberRepository.findById(savedMember.getId());
+        List<Favorite> favorites = memberOptional.get().getFavorites();
+
+        // then
+        favorites.stream().forEach(favorite -> {
+            assertThat(favorite.getArrivalStation().getName())
+                    .containsPattern(Pattern.compile(regex));
+        });
```

Comment:
정규식은 검증하는 조건이 명확히 드러나지 않기 때문에 아래와 같이 명확하게 의도가 드러나도록 작성해보면 어떨까요?

* https://github.com/assertj/assertj-examples/tree/main/assertions-examples/src/test/java/org/assertj/examples

```suggestion
        Assertions.assertThat(favorites)
                .extracting(Favorite::getArrivalStation)
                .containsExactly(station2, station1)

        // 또는
                        
        Assertions.assertThat(favorites).containsExactly(favorite1, favorite2)
```